ote_sdk/ote_sdk/usecases/exportable_code/demo/README.md

```diff
@@ -0,0 +1,58 @@
+# Exportable code - demo package
+
+Demo package contains simple demo to get and visualize result of model inference.
+Name of package is a name of model which was deployed.
+
+## Structure of generated package:
+
+openvino.zip:
+
+  * [model](../model)
+    * [model.xml](../model/model.xml)
+    * [model.bin](../model/model.bin)
+  * [python](.)
+    * [README.md](./README.md)
+    * [demo.py](./demo.py)
+    * [requirements.txt](./requirements.txt)
+    * <model_name>-0.0-py3-none-any.whl
+
+
+## Prerequisites
+* Python 3.6+
```

Comment:
Not sure if OTE SDK has been verified to be compliant with Python 3.6. We use Python 3.8 in our CI pipelines. I'd propose to set Python 3.8+ as a requirement here for now, to be on a safe side.

@Ilya-Krylov FYI

Reply:
Updated version